src/Analyzers/MSTest.Analyzers/Resources.resx

```diff
@@ -117,6 +117,40 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="AssemblyInitializeShouldBeValidDescription" xml:space="preserve">
+    <value>Methods marked with [AssemblyInitialize] should follow the following layout to be valid:
+- it should be 'public' 
+- it should be 'static'
+- it should not be generic
+- it should take one parameter of type 'TestContext'
+- return type should be 'void', 'Task' or 'ValueTask'
+- it should not be 'async void'
+- it should not be a special method (finalizer, operator...).</value>
+  </data>
+  <data name="AssemblyInitializeShouldBeValidMessageFormat_NotAsyncVoid" xml:space="preserve">
+    <value>AssemblyInitialize Method '{0}' should return 'void', 'Task' or 'ValueTask'</value>
```

Comment:
You've been using lower cases "method" in the TestInit/Cleanup messages, let's do something similar here:

```suggestion
    <value>AssemblyInitialize method '{0}' should return 'void', 'Task' or 'ValueTask'</value>
```

Please update the other entries.